iam/api-client/service_account_keys.py

```diff
@@ -22,23 +22,31 @@
 """
 
 import argparse
+# [START iam_create_key]
+# [START iam_list_keys]
+# [START iam_delete_key]
 import os
 
 from google.oauth2 import service_account
 import googleapiclient.discovery
 
-credentials = service_account.Credentials.from_service_account_file(
-    filename=os.environ['GOOGLE_APPLICATION_CREDENTIALS'],
-    scopes=['https://www.googleapis.com/auth/cloud-platform'])
-service = googleapiclient.discovery.build(
-    'iam', 'v1', credentials=credentials)
+# [END iam_create_key]
+# [END iam_list_keys]
+# [END iam_delete_key]
 
 
 # [START iam_create_key]
 def create_key(service_account_email):
     """Creates a key for a service account."""
 
     # pylint: disable=no-member
```

Comment:
I'm not sure this disable=no-member directive is required anymore. Could you please delete it (in all cases) and re-run lint and see if an error occurs? If deleting it does cause an error to occur, we can re-add it.

Reply:
Will do!